requirements/constraints.txt

```diff
@@ -19,9 +19,9 @@ beautifulsoup4==4.9.1 \
 billiard==3.6.3.0 \
     --hash=sha256:bff575450859a6e0fbc2f9877d9b715b0bbc07c3565bb7ed2280526a0cdf5ede \
     --hash=sha256:d91725ce6425f33a97dfa72fb6bfef0e47d4652acd98a032bd1a7fbf06d5fa6a
-kombu==4.6.8 \
-    --hash=sha256:598e7e749d6ab54f646b74b2d2df67755dee13894f73ab02a2a9feb8870c7cb2 \
-    --hash=sha256:2d1cda774126a044d91a7ff5fa6d09edf99f46924ab332a810760fe6740e9b76
+kombu==4.6.9 \
```

Comment:
These packages have to be upgraded at the same time. Here are the Kombu changes:

https://kombu.readthedocs.io/en/master/changelog.html#version-4-6-9

These claim that 4.5 is the release series and 4.6 is the development series.